### DISCUSSION TITLE
Unable to run any remote desktop services like Xorg, VNC etc.

### DISCUSSION THREAD

**Discussion by muhammadmahadazher** [Joshua-Riek/ubuntu-rockchip#1112] [Q&A]
Hi! im having same issues i cant really run any of the remote protocols in ubuntu desktop 24.0.1 LTS presinsatlled orange pi 5 pro version all the services are running xorg, rdp, x2go, vnc etc. but when I actually connect them its just a blank black screen! 
can anyone help why?

**Comment by Bahazzie** [comment]
Wayland and Xorg don't go together, you have to disable Wayland for Xorg to work. It's also what I did. Then RealVNC will work like a charm. The other services like RDP I have not tested. Wayland is more futureproof, as soon as RealVNC works on Wayland I will switch back to it.
